openapi/weaviate/Ballerina.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+license = ["Apache-2.0"]
+keywords = ["Vector", "Search", "Cost/Freemium", "AI/Weaviate"]
+org = "ballerinax"
+name = "weaviate"
+version = "1.0.0"
+distribution = "2201.4.1"
+repository = "https://github.com/ballerina-platform/ballerinax-openapi-connectors"
```

Comment:
Add the icon here also

Reply:
Addressed in https://github.com/ballerina-platform/openapi-connectors/pull/754/commits/4011e12c5729754ff4de72e28289bb1362275fd0.